libr/cons/input.c

```diff
@@ -661,10 +661,15 @@ R_API bool r_cons_yesno(int def, const char *fmt, ...) {
 	va_end (ap);
 	fflush (stderr);
 	r_cons_set_raw (true);
-	(void)read (0, &key, 1);
-	write (2, " ", 1);
-	write (2, &key, 1);
-	write (2, "\n", 1);
+	char buf[] = " ?\n";
```

Comment:
This part should be tested better, I think, if it affects the logic (just in case).